plugins/systemstat.koplugin/main.lua

```diff
@@ -1,6 +1,9 @@
+
```

Comment:
Ah, interesting. Do they have public coding style guides like Google? (Stuff like https://google.github.io/styleguide/cppguide.html and https://google.github.io/styleguide/shell.xml)